src/RefTypes.js

```diff
@@ -0,0 +1,4 @@
+type RefHandler = (?HTMLElement) => void;
+type RefObject = { current?: HTMLElement};
+
+export type Ref = RefHandler | RefObject;
```

Comment:
Why can't we use `React$Ref<HTMLElement>` here?

Reply:
[`React$Ref<HTMLElement>`](https://flow.org/en/docs/react/types/#toc-react-ref) includes string refs which are legacy and react-popper does not support.